onnxruntime/python/tools/transformers/models/longformer/benchmark_longformer.py

```diff
@@ -222,23 +244,28 @@ def test_ort_memory(
     num_threads,
 ):
     print(
-        f"Testing memory for model={onnx_model_path}, batch_size={batch_size}, sequence_length={sequence_length}, global_length={global_length}, test_times={test_times}, num_threads={num_threads}"
+        f"Testing memory for model={onnx_model_path}, batch_size={batch_size}, sequence_length={sequence_length}, "
+        f"global_length={global_length}, test_times={test_times}, num_threads={num_threads}"
     )
 
     def inference():
+        # Update Arena strategy so that we can measure the mininum memory required
+        cuda_provider_options = {"arena_extend_strategy": "kSameAsRequested"}
```

Comment:
How much does it affect the latency compared to the default?

Reply:
It does not impact latency if proper warm up is used. For example, use a longest input to warm up, then there is no memory allocation for the following queries.